primefaces/src/main/resources/META-INF/primefaces.taglib.xml

```diff
@@ -30290,6 +30290,14 @@
             <required>false</required>
             <type>java.lang.String</type>
         </attribute>
+        <attribute>
```

Comment:
Don't forget to also add this to `/docs/16_0_0/treetable.md` and in the `whatsnew.md` for 16